index.js

```diff
@@ -49,6 +51,24 @@ function HomeAssistantPlatform(log, config, api){
       console.log("Plugin - DidFinishLaunching");
     }.bind(this));
   }
+
+  var es = new EventSource(config.host + '/api/stream?password=' + encodeURIComponent(this.password));
```

Comment:
This is supposed to be `api_password`. Also, can you use the `X-HA-access` header?